frontend/src/config.ts

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const axiosConfig = axios.create({
```

Comment:
This config does not seem to be referenced anywhere else apart from this file. So I doubt it's working...
Please see if [this](https://bobbyhadz.com/blog/axios-change-base-url#:~:text=%23%20Setting%20the%20axios%20Base%20URL%20by%20creating%20an%20instance,'%2C%20%7D)%3B%20placeholderApi%20.) resource helps